src/ViewerComponents/Image/ImagesContainer.svelte

```diff
@@ -112,36 +142,53 @@
      * @returns {boolean}
      */
     function checkStillImage(data) {
-        if (data && Array.isArray(data)) {
-            return data.every(obj => obj['knora-api:hasStillImageFileValue']);
+        if (data.hasOwnProperty('@graph') && Array.isArray(data['@graph'])) {
+            return data['@graph'].every(obj => obj['knora-api:hasStillImageFileValue']);
         } else {
-            return false;
+            return data.hasOwnProperty('knora-api:hasStillImageFileValue');
         }
     }
 
+    /**
+     * Wraps the data in an array if there is one or no result.
+     *
+     * @param data
+     */
+    function wrapData(data) {
+        if (data.hasOwnProperty('@graph')) {
+            return data['@graph'];
+        } else if (data['@id']) {
+            return [data];
+        } else {
+            return [];
+        }
+    }
 </script>
 
 {#if requestInfos}
     <div class="container">
-        {#await promise}
-            <Loading/>
+        {#await promise_data}
+            <Loading loading_text="...searching"/>
```

Comment:
These texts probably should be translated for other languages right?

Reply:
I haven't given any thoughts to internalization yet. This is a bigger construction area that we have to tackle separately in the future. There is Svelte i18n that we should consider for this task (https://lokalise.com/blog/svelte-i18n/). For the moment I leave it that way. Thank you.